### PR TITLE
New Module: Using Enclaves with Amazon EKS

### PR DESCRIPTION
**Pull request by berry2012**
*Issue #23 

*Description of changes:*
Initial content - New Module: Using Enclaves with Amazon EKS

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/8-infrastructure-security/2-using-enclaves/2-create-node/index.en.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+
+```bash
+MIME-Version: 1.0
+Content-Type: multipart/mixed; boundary="==MYBOUNDARY=="
+--==MYBOUNDARY==
+Content-Type: text/x-shellscript; charset="us-ascii"
+#!/bin/bash -e
+readonly NE_ALLOCATOR_SPEC_PATH="/etc/nitro_enclaves/allocator.yaml"
+# Node resources that will be allocated for Nitro Enclaves
+readonly CPU_COUNT=4
```
Comment (vpodugu) [inline content/8-infrastructure-security/2-using-enclaves/2-create-node/index.en.md:65]: Ask user to enter the value.

File: content/8-infrastructure-security/2-using-enclaves/2-create-node/index.en.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+readonly MEMORY_MIB=768
```
Comment (vpodugu) [inline content/8-infrastructure-security/2-using-enclaves/2-create-node/index.en.md:66]: Ask user to enter the value.

File: content/8-infrastructure-security/2-using-enclaves/index.en.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+title : "Using Enclaves with Amazon EKS"
+weight : 31
+---
+
+[AWS Nitro Enclaves](https://aws.amazon.com/ec2/nitro/nitro-enclaves/) is an [Amazon EC2](https://aws-content-sandbox.aka.amazon.com/ec2/) capability that enables customers to create isolated compute environments to further protect and securely process highly sensitive data within their EC2 instances. Nitro Enclaves helps customers reduce the attack surface area for their most sensitive data processing applications. AWS Nitro Enclaves supports [Amazon EKS](https://aws.amazon.com/eks/) and Kubernetes for orchestrating Nitro enclaves. Customers can use familiar Kubernetes tools to orchestrate, scale, and deploy enclaves from a Kubernetes pod.
```
Comment (vpodugu) [inline content/8-infrastructure-security/2-using-enclaves/index.en.md]: Remove the link for Amazon EC2 
Remove the link for Amazon EKS

File: content/8-infrastructure-security/2-using-enclaves/index.en.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+
```
Comment (vpodugu) [inline content/8-infrastructure-security/2-using-enclaves/index.en.md:7]: Add a visual representation.

**Review by vpodugu — CHANGES_REQUESTED** [review]
Please fix these.

**Review by allamand — APPROVED** [review]
As discuss let's merge it

**Pull request merged**